Zend/zend_language_parser.y

```diff
@@ -701,8 +701,8 @@ class_statement_list:
 class_statement:
 		variable_modifiers property_list ';'
 			{ $$ = $2; $$->attr = $1; }
-	|	T_CONST class_const_list ';'
-			{ $$ = $2; RESET_DOC_COMMENT(); }
+	|	method_modifiers T_CONST class_const_list ';'
+			{ $$ = $3; $$->attr = $1; RESET_DOC_COMMENT(); }
```

Comment:
`method_modifiers` allows static, abstract and final as well. Either we should introduce a separate production for only public/protected/private, or we have to forbid other flags in zend_compile.

Also the `RESET_DOC_COMMENT()` can be dropped now.